File: Easy/26-remove-duplicates-from-sorted-array/try-11.py
/*
 * Problem #26: Remove Duplicates from Sorted Array
 * Difficulty: Easy
 * Submission: Try 11
 * status: Accepted
 * Language: python3
 * Date: 30/11/2024, 12:32:48
 * Link: https://leetcode.com/problems/remove-duplicates-from-sorted-array/
 */

class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        j=0
        for i in range(len(nums)):
            if(nums[j]!=nums[i]):
                nums[j:=j+1]=nums[i]
        return j+1

File: Easy/242-valid-anagram/try-11.cpp
/*
 * Problem #242: Valid Anagram
 * Difficulty: Easy
 * Submission: Try 11
 * status: Accepted
 * Language: cpp
 * Date: 06/12/2024, 15:49:41
 * Link: https://leetcode.com/problems/valid-anagram/
 */

#include<algorithm>

class Solution {
public:
    bool isAnagram(string s, string t) {

        if(s.size()!=t.size()){
            return false;
        }
        
        sort(t.begin(),t.end());
        sort(s.begin(),s.end());
        return s==t;    
    }
        
};
